Fix bad link in Theme - distribution - exporting page (#570)
PrestaShop/docs: src/content/1.7/themes/distribution/exporting.md

it's on the disk.

> If you want to theme to appears in the backoffice, it's simply have to contain a `config/theme.yml` file.

Reviewer: ```suggestion
If you want the theme to appears in the backoffice, it simply has to contain a `config/theme.yml` file.
```

I've some doubt about my english, but it looks like there is some problems there too, I know it's not related to your PR but maybe we can improve this at the same time regarding the fact that you open some PRs to improve typos :)

This will only display it, if you want to select it as your active theme, it has to be valid. Read ["What

> makes a valid theme"]({{< ref "1.7/themes/distribution/testing/" >}}).

Reviewer 2: ```suggestion
makes a valid theme"]({{< ref "1.7/themes/distribution/testing.md" >}}).
```
